docs/extensions/datacollector-migration.md

```diff
@@ -0,0 +1,44 @@
+# Migrate your TestPlatform v0(TPv0)/TestPlatoformv1(TPv1) datacollector to TestPlatform v2(current version) datacollector
```

Comment:
Tpv0, TPv1 and TPv2 are internal terms. Data collectors have always been TPv0 based. Suggest using TMI or MSTest.exe.